lib/commandline/CommandLineText.cs

```diff
@@ -56,39 +56,39 @@ public static BaseSentenceBuilder CreateBuiltIn ()
         {
```

Comment:
I'm avoiding changing this file as far as I can. It was another library (which has now moved on in a way which isn't helpful to us) and I'd like to keep it as close to the original source as possible.